### DISCUSSION TITLE
How can i just use the Driver for showing only the Cell Voltages?

### DISCUSSION THREAD

**Discussion by DrknMonkey** [Louisvdw/dbus-serialbattery#490] [Q&A] [answered]
Hello,

first, thank you very much for this Driver.
Is it possible to use the Driver for only showing the Cell Voltages first? I have already set all "Values" on False, but in the Cerbo it shows me that the MPPT is still managed from the  BMS and not as Stand Alone. i have also reset the MPPT but it still works as Slave.

![Cerbo 1](https://user-images.githubusercontent.com/127410287/224042735-0f3c8815-0fb5-4799-9f99-c75dd32dd77a.JPG)
![Cerbo 2](https://user-images.githubusercontent.com/127410287/224042750-e252952e-bf31-44ad-9a07-34aed4adfe07.JPG)
![Cerbo 3](https://user-images.githubusercontent.com/127410287/224042756-6d561bc8-9a8a-410b-9cf3-7d59b41d98a1.JPG)
![Cerbo 4](https://user-images.githubusercontent.com/127410287/224042761-4d0bd2af-f9a4-4464-8439-fee64b14f00f.JPG)

**Comment by DrknMonkey** [comment]
It seems the Driver work correct and accept also my Settings for the Voltage or the Current, also the Settings that i have set to "False".
But still my MPPT is controlled external by the BMS... any Idea?

![Putty 1](https://user-images.githubusercontent.com/127410287/224846130-72f8f9ce-3dce-4968-97be-083bbae86399.JPG)

![Putty 2](https://user-images.githubusercontent.com/127410287/224846133-8386caa8-d1c2-43c1-b236-c568c7f966ef.JPG)

**Comment by mr-manuel** [comment]
It's not possible.

https://github.com/Louisvdw/dbus-serialbattery/discussions/492
https://github.com/Louisvdw/dbus-serialbattery/discussions/482

**Comment by mr-manuel** [comment, marked as answer]
I have a update for you.

The battery monitor is not the same as the BMS selected for DVCC.

With Venus OS >3.00 you can select the BMS that should be used for DVCC:
![grafik](https://user-images.githubusercontent.com/12592765/232209770-4a46e04e-bb2f-4afd-b42b-f3da7e0b63d6.png)

![grafik](https://user-images.githubusercontent.com/12592765/232209979-a64e9dc6-a6d2-45f4-bb39-cdb05ba1dcf9.png)


Source: https://community.victronenergy.com/questions/199460/multiple-battery-monitor-available-only-selected-a.html?childToView=199503#comment-199503

**Comment by DrknMonkey** [comment]
Thank you very much, i will test it ;)
Ok, for me is no newer Firmware as v2.94 available... maybe later, then i will try it.
